src/node_http2.h

```diff
@@ -487,13 +473,17 @@ class Http2Session : public AsyncWrap,
     return stream_buf_;
   }
 
+  void Close() override;
+
  private:
   StreamBase* stream_;
   StreamResource::Callback<StreamResource::AllocCb> prev_alloc_cb_;
   StreamResource::Callback<StreamResource::ReadCb> prev_read_cb_;
   padding_strategy_type padding_strategy_ = PADDING_STRATEGY_NONE;
 
   char stream_buf_[kAllocBufferSize];
+
+  uv_prepare_t* prep_ = nullptr;
```

Comment:
For reasons of mechanical sympathy, I'd put it before the 64 kB `stream_buf_` buffer.